tools/tracy/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.13...3.18 FATAL_ERROR)
+project(inviwo-tracy DESCRIPTION "Inviwo tracy wrapper" LANGUAGES CXX)
+
+set(HEADER_FILES
+    include/inviwo/tracy/tracy.h
```

Comment:
I can’t see the difference between these ones and https://github.com/wolfpld/tracy

Why not submodule?

Reply:
we need to replicate that to avoid having to include tracy all the time. I also want the macro to follow our style.

We obviously don't want to add any more submodule, that is a really bad solution.

Reply:
Ok, then please add a comment on where it comes from so that we can update it in the future in case it changes. I saw that they also have opencl support, but it might not be worthwhile to add?

Reply:
OpenCL would probably be nice, I have not had the need for it yet, but it should be very easy to add

Reply:
there is also vulkan i think when that becomes relevant...